.github/workflows/end2end.yaml

```diff
@@ -428,6 +428,19 @@ jobs:
         env:
           STAGE: end2end-http
         if: always()
+      - name: "Debug: SSH to runner"
```

Comment:
not part of this PR, should be in https://scality.atlassian.net/browse/ZENKO-4749

Reply:
Yes , all of the debug related commits will be dropped